### ISSUE TITLE
Deepth information fron camera

### ISSUE THREAD

**Issue by EnricoBeltramo** [victorprad/InfiniTAM#137]
I'm very novel to DTAM algorithm and I'm still understanding it. For my research I need to have a 3D reconstruction from a normal RGB camera, this algorithm is able to that or need always the deepth measurement from camera?

**Comment by sgolodetz** [comment]
InfiniTAM doesn't implement DTAM, it implements a hashing-based variant of KinectFusion. You need depth to make it work, but the depth doesn't need to come directly from a depth camera (e.g. you could try a mono depth estimation approach).

**Comment by mpottinger** [comment]
@EnricoBeltramo Yes I believe that is the case. There are algorithms to get a depth map from a monocular rgb camera, such as monodepth. 

However I am not sure if the accuracy is sufficient.
